interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -2081,7 +2081,7 @@ protected void assertX500SubjectDn(String tlsInfo) {
   }
 
   protected int operationTimeoutMillis() {
```

Comment:
Oh, it is defined in a protected method. Would you mind changing this just in the interop client binary?

https://github.com/grpc/grpc-java/blob/69d4e71309c64573b508c936f96ace3b1d016a59/interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java#L490

Having it here is _fine_, and I'd be willing to approve it. But having it in the other seems more precise.